SLE/wizard/style.qss

```diff
@@ -41,4 +41,7 @@
   margin: 0px;
 }
 
+#DialogHeadingTop {
+  font: 18pt;
+}
```

Comment:
If we compare with installation.qss, we see that there are `#DialogHeadingLeft` and `#DialogHeadingTop` there. We should fix it similarly here. A couple of lines above there is `#DialogHeading`.

This is a fallout of repositioning the header for the installation, https://bugzilla.suse.com/show_bug.cgi?id=868859